### PR TITLE
NTBS-2302 Add disease site list to Record_CaseData

### PR DESCRIPTION
**Pull request by benjimarshall**
### Description
Add a comma separated list of disease sites to `Record_Data.DiseaseSiteList`, a new field. This comes from NTBS and ETS
depending on the notification, and uses disease site names from the respective system.

This is based on how a similar field is calculated for `ForestExtract`. 

### Testing
Tested the insertion locally using the basic test data, and have tested the queries work on live data (just not the insertion bit). This code has not been deployed anywhere

### CODE REVIEW

**Review by aptemus — CHANGES_REQUESTED** [review]
@benjimarshall Just one suggested change (albeit in two places)

File: source/dbo/Stored Procedures/Power BI Reporting/uspGenerateEtsCaseRecord.sql [hunk 1]
```diff
@@ -1,6 +1,19 @@
...
+	DECLARE @TempDiseaseSites TABLE
+	(
+		TuberculosisEpisodeId uniqueidentifier,
+		[Description] nvarchar(2000)
+	);
+
+	INSERT INTO @TempDiseaseSites
+	SELECT TuberculosisEpisodeId, [Description] = STRING_AGG([Name], N', ')
+		FROM [$(ETS)].dbo.TuberculosisEpisodeDiseaseSite diseaseSite
+			INNER JOIN [$(ETS)].dbo.DiseaseSite sites ON sites.Id = diseaseSite.DiseaseSiteId
```
[the remark this answers is not in the document]
Comment (benjimarshall) [reply source/dbo/Stored Procedures/Power BI Reporting/uspGenerateEtsCaseRecord.sql:15]: Yep, that makes sense, done!

File: source/dbo/Stored Procedures/Power BI Reporting/uspGenerateNtbsCaseRecord.sql [hunk 2]
```diff
@@ -1,6 +1,18 @@
...
+	DECLARE @TempDiseaseSites TABLE
+	(
+		NotificationId int,
+		[Description] nvarchar(2000)
+	);
+
+	INSERT INTO @TempDiseaseSites
+	SELECT NotificationId, [Description] = STRING_AGG([Description], N', ')
+		FROM [$(NTBS)].[dbo].[NotificationSite] notificationSite
+		JOIN [$(NTBS)].[ReferenceData].[Site] sites ON notificationSite.SiteId = sites.SiteId
```
[the remark this answers is not in the document]
Comment (benjimarshall) [reply source/dbo/Stored Procedures/Power BI Reporting/uspGenerateNtbsCaseRecord.sql]: Done!

**Review by JoshCadney — APPROVED** [review]
Looks good :-)

**Pull request merged**